### PR TITLE
Fix #7 XML comments

### LINKED ISSUE #7: Manage HTML/XML comments
And especially our `<!-- more -->` tag. cc @0nko

### PR DESCRIPTION
**Pull request by 0nko**
Fix #7 - handle XML comments

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -15,6 +15,15 @@
                       android:layout_width="wrap_content"
                       android:layout_height="wrap_content">
 
+            <ImageButton android:id="@+id/html"
```
Comment (maxme) [inline app/src/main/res/layout/activity_main.xml]: Can we use a text button for now?

File: app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt [hunk 2]
```diff
@@ -137,6 +137,17 @@ class MainActivity : Activity() {
         }
     }
 
+    private fun setupHtml() {
+        val clear = findViewById(R.id.html) as ImageButton
```
Comment (maxme) [inline app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt]: Nit: s/clear/html/

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (0nko) [inline app/src/main/res/layout/activity_main.xml]: Sure.

**Comment by 0nko** [comment]
Updated as per comments.

**Comment by maxme** [comment]
:shipit:

**Pull request merged**